frontend/site-registry/src/app/features/cart/CartSlice.ts

```diff
@@ -105,17 +105,41 @@ const cartSlice = createSlice({
         state.fetchRequestStatus = RequestStatus.loading;
       })
       .addCase(fetchCartItems.fulfilled, (state, action) => {
-        state.fetchRequestStatus = RequestStatus.success;
-        state.cartItems = action.payload;
+        if (
+          action.payload &&
+          action.payload.data &&
+          action.payload.data.getCartItemsForUser &&
+          action.payload.data.getCartItemsForUser.httpStatusCode === 200
```

Comment:
Can this be replaced with optional chaining operators instead?
```typescript
action.payload?.data?.getCartItemsForUser?.httpStatusCode === 200
 ```

Reply:
Addressed this comment as part of PR https://github.com/bcgov/nr-site-registry/pull/60